File: dotnet-advanced/Config/AppConfig.cs
using System;

namespace AdvancedUserService.Config
{
    public static class AppConfig
    {
        public const string ServiceName = "advanced-user-service-dotnet";
        public const string Version = "1.0.0";

        public static string Environment =>
            System.Environment.GetEnvironmentVariable("ENV") ?? "development";

        public static class Cors
        {
            public const bool Enabled = true;
            public const string Origins = "*";
        }
    }
}

File: java-advanced/src/main/java/functions/config/AppConfig.java
package functions.config;

public class AppConfig {
    public static final String SERVICE_NAME = "advanced-user-service-java";
    public static final String VERSION = "1.0.0";
    public static final String ENVIRONMENT = System.getenv().getOrDefault("ENV", "development");

    public static class CORS {
        public static final boolean ENABLED = true;
        public static final String ORIGINS = "*";
    }
}
